external_search.py

```diff
@@ -562,6 +563,8 @@ def _f(s):
                 # no longer happen but we'll handle it.
                 genre_ids = [genre.id for genre in genres]
             clauses.append(dict(terms={"genres.term": genre_ids}))
+        if customlist_ids is not None:
+            clauses.append(dict(terms={"list_id": customlist_ids}))
```

Comment:
This looks good to me although I'm not 100% sure how it will handle an empty list.